contracts/mock/MockEscrowedLine.sol

```diff
@@ -1,89 +1,68 @@
 pragma solidity 0.8.9;
 
-import  {EscrowedLine} from "../modules/credit/EscrowedLine.sol";
-import { IEscrowedLine } from "../interfaces/IEscrowedLine.sol";
+import {EscrowedLine} from "../modules/credit/EscrowedLine.sol";
+import {IEscrowedLine} from "../interfaces/IEscrowedLine.sol";
 import {LineOfCredit} from "../modules/credit/LineOfCredit.sol";
 import {LineLib} from "../utils/LineLib.sol";
 import {IEscrow} from "../interfaces/IEscrow.sol";
 import {CreditLib} from "../utils/CreditLib.sol";
 import {ILineOfCredit} from "../interfaces/ILineOfCredit.sol";
 
 contract MockEscrowedLine is EscrowedLine, LineOfCredit {
-    
-    
-    constructor(
-        address _escrow,
-        address oracle_,
-        address arbiter_,
-        address borrower_,
-        uint ttl_
-        ) EscrowedLine(_escrow)
-    LineOfCredit(oracle_, arbiter_, borrower_, ttl_){
-        
-    }
-    
-
-  function _init() internal override(EscrowedLine, LineOfCredit) returns(LineLib.STATUS) {
-    return EscrowedLine._init();
-  }
+    constructor(address _escrow, address oracle_, address arbiter_, address borrower_, uint256 ttl_)
```

Comment:
smh i hate formatters so much lol. Its way better to keep these as separate lines but oh well